ocamltest/ocamltest_stdlib.ml

```diff
@@ -84,22 +84,36 @@ end
 module Sys = struct
   include Sys
 
-  let run_system_command prog args =
-    let command = Filename.quote_command prog args in
-    match Sys.command command with
-    | 0 -> ()
-    | _ as exitcode ->
-      Printf.eprintf "System command %s failed with status %d\n%!"
-        command exitcode;
-      exit 3
-
-  let mkdir dir =
-    if not (Sys.file_exists dir) then
-      run_system_command "mkdir" [dir]
+  let erase_file path =
+    try Sys.remove path
+    with Sys_error _ when Sys.win32 && Ocamltest_config.libunix <> None ->
+      (* Deal with read-only attribute on Windows. Ignore any error from chmod
+         so that the message always come from Sys.remove *)
+      let () = try Unix.chmod path 0o666 with Sys_error _ -> () in
+      Sys.remove path
+
+  let rm_rf path =
+    let rec erase path =
+      if Sys.is_directory path
+      then Array.iter (fun entry -> erase (Filename.concat path entry))
+                      (Sys.readdir path)
+      else erase_file path
```

Comment:
I just noticed that ocamltest  leaves behind lots of empty directories under `_ocamltest/`, and this code could be the reason why.  What about:
```
    let rec erase path =
      if Sys.is_directory path then begin
        Array.iter (fun entry -> erase (Filename.concat path entry))
                   (Sys.readdir path);
        Sys.rmdir path   (* <---- NEW *)
      end
      else erase_file path
```